sql/moz-fx-data-shared-prod/telemetry_derived/newtab_daily_interactions_aggregates_v1/query.sql

```diff
@@ -0,0 +1,83 @@
+SELECT
+  submission_date,
+  search_engine,
+  search_access_point,
+  pocket_story_position,
+  default_private_search_engine,
+  default_search_engine,
+  os,
+  os_version,
+  country_code,
+  channel,
+  browser_version,
+  browser_name,
+  newtab_open_source,
+  pocket_is_signed_in,
+  pocket_enabled,
+  pocket_sponsored_stories_enabled,
+  topsites_enabled,
+  newtab_homepage_category,
+  newtab_newtab_category,
+  newtab_search_enabled,
+  topsites_rows,
+  is_new_profile,
+  activity_segment,
+  COUNT(DISTINCT client_id) AS client_count,
```

Comment:
This data will be an issue since it'll be quite common to have a client doing interactions across different dimensions in a day. But it doesn't seem feasible to create tables for each so we may need to just double check is stakeholders if there are any specific ones that really require no double counting.

Reply:
I agree. If you feel by having it is confusing, I can remove these count(distinct --) fields from that table:
- client_count
- legacy_telemetry_client_count
- newtab_interactions_visits

Reply:
I think we need to keep it as that's the closes we have to what is currently in the dashboard. It'll just be overcounting in certain cases once we SUM.